docs/farming-&-staking/farming/pulsar.mdx

```diff
@@ -40,9 +40,9 @@ We are grateful for your adaptability and support as we navigate this change tog
 
 :::note Learn More About Syncing, Plotting & Farming
 
-For more information about the principles of farming on the Subspace Network, you can still refer to the Subnomicon links below:
-- [Synchronization](https://subnomicon.subspace.network/docs/network/network_protocols/#synchronization)
-- [Plotting](https://subnomicon.subspace.network/docs/consensus/consensus/plotting)
-- [Farming](https://subnomicon.subspace.network/docs/consensus/consensus/farming)
+For more information about the principles of farming on the Subspace Network, you can still refer to the Autonpmys Academy links below:
+- [Synchronization](https://academy.autonomys.net/subspace-protocol/network-architecture/networking-protocols#synchronization)
```

Comment:
These bullets look better for the list of links. Maybe apply this approach to the lists of links everywhere in this update.